docs/components/HeaderMenu.js

```diff
@@ -35,7 +33,7 @@ export default function HeaderMenu({ isHeader }: {| isHeader?: boolean |}): Node
         </Text>
       </Tooltip>
 
-      <Tooltip inline text="Check out the source code on GitHub">
+      <Tooltip inline text="Check out the source code on GitHub" zIndex={popoverZIndex}>
```

Comment:
Super nit, but we may want to change the tooltip text to read, "View source code" to avoid the amount of text wrapping the current tooltip has
<img width="447" alt="Screen Shot 2022-05-25 at 10 38 49 AM" src="https://user-images.githubusercontent.com/50343812/170328577-0ced5fa1-d516-4477-b174-4ae11826cac0.png">
.

Reply:
ahh good call